docs/usage.md

```diff
@@ -1,89 +1,118 @@
-# CodeChecker command line examples
-### 1. Quickly check some files and print results in command line
-```
-CodeChecker quickcheck -b make
-```
+# CodeChecker HOWTO
 
-### 2. Check a project, update earlier results  and view results from a browser
-Runs make, logs build, run analyzers and store the results in sqlite db.
-```
-CodeChecker check -b make
+This is lazy dog howto to using CodeChecker analysis.
+It invokes clang static analyzer and clang tidy tools to analyze your code.
+
+## Step1: Inegrate CodeChecker into your build system
+Codechecker only analyzes what is also built by your build system. As a first step 
+
+1. Select a module to build (open source tmux in this example).
 ```
-  
-Start webserver, which listens on default port localhost:8001 
-results can be viewed in a browser
+cd tmux
+./configure
 ```
-CodeChecker server
+2. Clean that module. e.g. '''make clean'''
```

Comment:
`'''` does not make sense as any formatting string in Markdown, and it does not belong here, if it's a line below too.